README.md

```diff
@@ -71,7 +76,7 @@ StartLimitIntervalSec=0
 Type=simple
 Restart=always
 RestartSec=1
-ExecStart=node /path/to/boarding-duck/bot.js --config /path/to/boarding-duck/config.json
+ExecStart=cd /path/to/boarding-duck/ && npm run start:setup
```

Comment:
I think there is a way of specifying a working directory, instead of changing it from the command, non ?